openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -0,0 +1,147 @@
+"""
+UserPartitionScheme for enrollment tracks.
+"""
+from courseware.masquerade import (  # pylint: disable=import-error
+    get_course_masquerade,
+    get_masquerading_group_info,
+    is_masquerading_as_specific_student,
+)
+from course_modes.models import CourseMode, EnrollmentMode
+from student.models import CourseEnrollment
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.verified_track_content.models import VerifiedTrackCohortedCourse
+from xmodule.partitions.partitions import NoSuchUserPartitionGroupError, Group, UserPartition
+
+
+# EnrollmentMode IDs will be low integers (starting at 0), which is important so that they
+# do not overlap with Groups in the CohortUserPartition or the RandomUserPartitionScheme
+# (CMS' course_group_config uses a minimum value of 100 for all generated IDs).
+ENROLLMENT_GROUP_IDS = {}
+try:
+    for enrollment_mode in EnrollmentMode.objects.all():
+        ENROLLMENT_GROUP_IDS[enrollment_mode.mode_slug] = enrollment_mode.id
+except:  # pylint: disable=bare-except
+    pass
+
+if not ENROLLMENT_GROUP_IDS:
```

Comment:
Once again, can we move this to `lms/env/test.py` instead, so it's not part of production code?